addons/payment/static/src/js/post_processing.js

```diff
@@ -1,10 +1,7 @@
 /** @odoo-module **/
+'use strict';
```

Comment:
`use strict` is already by default in Odoo, so no need to add it here.

Reply:
I thought it was only automatic for new-style modules (cfr https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#old-style-module-guidelines), but maybe that doc is outdated? WDYT?

Reply:
Discussed with JS team. Outcome: not needed.